packages/spindle-ui/src/Pagination/hooks/useShowItem.test.ts

```diff
@@ -1,44 +1,601 @@
 import { renderHook } from '@testing-library/react-hooks';
 import { useShowItem } from './useShowItem';
 
+// ページ総数の閾値
+const TOTAL_THRESHOLD = 100;
+// 表示出来る数字(アイテム)の最大数
+const MAX_SHOW_ITEM_SIZE = 5;
+
 describe('useShowItem()', () => {
-  it('should return show props', () => {
-    const current = 8;
-    const total = 20;
+  it('use show item current 1 total 1', () => {
+    const current = 1;
+    const total = 1;
 
-    const { result } = renderHook(() => useShowItem({ current, total }));
+    const { result } = renderHook(() =>
+      useShowItem({
+        current,
```

Comment:
imo. テスト多くなることが予想されるので、constに置かず直接渡して行数削減してもらってよさそうです:pray:
```suggestion
        current: 1,
```